doc/sphinx-guides/source/developers/dependencies.rst

```diff
@@ -427,14 +431,16 @@ The codebase is structured like this:
 .. code-block::
 
     <project root>              # Dataverse WAR Module
+    ├── pom.xml                 # (POM file of WAR module)
     ├── modules                 #
     │   └── dataverse-parent    # Dataverse Parent Module
-    ├── pom.xml                 # (POM file of WAR module)
+    │       └── pom.xml         # (POM file of Parent Module)
     └── scripts                 #
         └── zipdownload         # Zipdownloader JAR Module
+            └── pom.xml         # (POM file of Zipdownloader Module)
```

Comment:
Yes, I put the root `pom.xml` at the top even though Unix wouldn't sort it that way. I'm going for readability. 😄